resources/git-fork.md

```diff
@@ -0,0 +1,98 @@
+# Git: Forking a repository
+
+> _Ah, the life! Branching, committing, pushing, pulling, merging... modifying code to my hearts' content... breathing life into features like some sort of magician or squashing bugs like an exterminator. Bliss. Until..._
+
+```bash
+! [remote rejected] master -> master (protected branch hook declined)
+error: failed to push some refs to 'git@github.com:some-account/some-repo.git'
+```
+
+☹️
+
+When working with code, there are times when you will want to modify code that resides in a repository that is not under your control, or with which for a variety of reasons you don't wish to interact. You might want to introduce features that are outside the scope of the project. You may want to build something completely unrelated and just use the repository as a starting point. Maybe a useful library needs a bug fix, but the maintainer has abandoned the repository. 
+
+For all these times and [more](#why-fork), there's **Forking**!
+
+## What is a Fork?
+
+Forking is creating a new copy of the entire repository. This might sound like `git clone`, and behind the scenes it is. The difference is in where the new copy resides. A "clone" typically refers to a _local_ copy of some repository. A _fork_, in contrast, is a second *remote* repository distinct from the original that it copies.
```

Comment:
you can probably leave out `that it copies` at the end of this sentence, since you mention that it's a copy above.